ldms/src/store/slurm/store_slurm.c

```diff
@@ -564,7 +566,9 @@ _open_store(struct sos_instance *si, ldms_set_t set)
 	}
 
  add_schema:
+	uuid_generate(schema_uuid);
 	slurm_schema_template.name = si->schema_name;
+	slurm_schema_template.uuid = (char*)schema_uuid;
```

Comment:
why is this cast necessary?

Reply:
Without the casting, GCC warned about "converts between pointers to integer types with different sign".